assignments/assignment_4/README.md

```diff
@@ -1,25 +1,15 @@
-# Assignment 4: Design Patterns
+# Assignment 4: OOP and VSCode Shortcuts
 
-The focus of this (final) assignment is about OOP and design patterns. And we have bad news: we went to check the Eurostat source and now they are using JSON as a data format. Also, the file was zipped.
+The focus of this assignment is about OOP. In particular, we want to make sure that we only provide valid values for our countries. This is the perfect scenario to make use of the Enum class. Note that this is a special case of classes since they are rarely instantiated. 
 
-> Don't forget to create a branch for this assignment.
-
-## 1- Design Patterns
-
-1. Copy the JSON data from the `assignment_4` folder to the `life_expectancy/data` folder.
-2. Refactor your code so that is can accept data in other formats if necessary. You can use the [Strategy pattern](https://refactoring.guru/design-patterns/strategy) for this, but you can also use a different one if you prefer.
-3. Ensure you still have complete test coverage and a high `pylint` score.
+But going beyond that, our goal is to create the most annoying assignment ever written. The twist is that there is an easy way out: VSCode shortcuts.
 
-## 2- Enums
-
-Ok, the final stretch. You can do this. Passing a country as a string is not very safe (imprecise types are a code smell). We can use an enum to make sure that we only pass valid countries.
-
-1. Create an `enum.Enum` called `Country` with possible country values.
-2. Then, modify the necessary functions and tests to accept a `Country` instead of a string. Don't forget the type hints.
-3. Finally, add a class method to `Country` that returns a list of all the _actual_ countries (so, it removes values like EU28, EFTA, etc). Add a test for this method.
+> Don't forget to create a branch for this assignment.
 
-## 3- Code Review
+## 1- Enums and Shortcuts
 
-As with the previous assignment, a peer should review your code and you will be reviewing the code of a fellow colleague.
+Passing a country as a string is not very safe (imprecise types are a code smell). We can use an enum to make sure that we only pass valid countries.
 
-Was your PR approved? Great! Now you can merge it into `main`, submit the assignment and go celebrate! 🎈
+1. Create an `enum.Enum` called `Country` with possible country values. The idea behind this step is to find the most appropriate VSCode shortcut/s that you learned about in the [text editor](../../07_text_editors/README.md) module. 
+2. Modify the necessary functions and tests to accept a `Country` instead of a string. Don't forget the type hints.
+3. Finally, add a class method to `Country` that returns a list of all the _actual_ countries (so, it removes values like EU28, EFTA, etc). Add a test for this method. Again, make use of as many shortcuts as possible, it will make your life much easier.
```

Comment:
```suggestion
2. Modify the necessary functions and tests to accept a `Region` instead of a string. Don't forget the type hints.
3. Finally, add a class method to `Region` that returns a list of all the _actual_ countries (so, it removes values like EU28, EFTA, etc). Add a test for this method. Again, make use of as many shortcuts as possible, it will make your life much easier.
```

Renamed "Country" to "Region" since "EU28", etc are not countries. It also makes the useness of  the `get_countries` method more explicit